firestore/integration_test_internal/src/validation_test.cc

```diff
@@ -472,6 +472,77 @@ TEST_F(ValidationTest,
   EXPECT_NE(Firestore::GetInstance("foo"), nullptr);
 }
 
+TEST_F(ValidationTest,
```

Comment:
This test is leaking memory, and could potentially interfere with other tests.

When using `Firestore::GetInstance(..)` directly, we need to destroy instances after test.